plugins/orchestrator-common/package.json

```diff
@@ -37,7 +37,8 @@
   ],
   "files": [
     "config.d.ts",
-    "dist"
+    "dist",
+    "src/generated/docs"
```

Comment:
once the issues with rebase have been resolved, i think this should be `docs/markdown`

Reply:
it can not be markdown. we expose the whole docs folder. we want to go to index.html

Reply:
i made it docs/html